code/modules/vehicles/vehicle_actions.dm

```diff
@@ -127,24 +127,17 @@
 	desc = "Honk your classy horn."
 	button_icon_state = "car_horn"
 	var/hornsound = 'sound/items/carhorn.ogg'
-	var/last_honk_time
 
 /datum/action/vehicle/sealed/horn/Trigger()
-	if(world.time - last_honk_time > 20)
-		vehicle_entered_target.visible_message("<span class='danger'>[vehicle_entered_target] loudly honks!</span>")
-		to_chat(owner, "<span class='notice'>You press the vehicle's horn.</span>")
-		playsound(vehicle_entered_target, hornsound, 75)
-		last_honk_time = world.time
-
-/datum/action/vehicle/sealed/horn/clowncar/Trigger()
-	if(world.time - last_honk_time > 20)
-		vehicle_entered_target.visible_message("<span class='danger'>[vehicle_entered_target] loudly honks!</span>")
-		to_chat(owner, "<span class='notice'>You press the vehicle's horn.</span>")
-		last_honk_time = world.time
-		if(vehicle_target.inserted_key)
-			vehicle_target.inserted_key.attack_self(owner) //The key plays a sound
-		else
-			playsound(vehicle_entered_target, hornsound, 75)
+	if(TIMER_COOLDOWN_CHECK(src, COOLDOWN_CAR_HONK))
+		return
+	TIMER_COOLDOWN_START(src, COOLDOWN_CAR_HONK, 2 SECONDS)
```

Comment:
Why timer cooldowns instead of normal ones?

Reply:
Because it's not a common cooldown